### PR TITLE
Dark/light theme

### PR DESCRIPTION
**Pull request by cupoftea4**
https://trello.com/c/9yiTE4Bu/4-client-configure-dark-light-theme

### CODE REVIEW

File: utils/theme.ts
Comment (Oleksii-Morozov) [inline utils/theme.ts:1]: File theme.ts in directory styles exist, please move code to correct file

File: utils/theme.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  background: 'white',
+  foreground: '#181818',
+  text: '#181818',
+  bgCircle: 'rgba(255, 190, 103, 1)',
+  modalBg: '#F3F3F3',
+  backdrop: 'rgba(32, 32, 32, 0.23)',
+  cardBg: '#FFFFFF',
+  stepDone: '#3D974B',
+  stepUndone: '#D5D5D5',
+
```
Comment (Oleksii-Morozov) [inline utils/theme.ts:28]: Why empty space?

File: utils/theme.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  foreground: '#F3F3F3',
+  text: '#F3F3F3',
+  bgCircle: 'rgba(37, 44, 34, 0.34)',
+  modalBg: '#181818',
+  backdrop: 'rgba(0, 0, 0, 0.27)',
+  cardBg: '#232323',
+  stepDone: '#3D974B',
+  stepUndone: '#6B6B6B',
+
+
```
Comment (Oleksii-Morozov) [inline utils/theme.ts:43]: Why empty space?

Comment (Oleksii-Morozov) [inline utils/theme.ts:43]: Why empty space?

File: utils/theme.ts (hunk 1 above)
Comment (Oleksii-Morozov) [inline utils/theme.ts:28]: Why empty space?

File: utils/theme.ts
Comment (Oleksii-Morozov) [inline utils/theme.ts:1]: Please move to styles/theme.ts

**Review by Oleksii-Morozov — CHANGES_REQUESTED** [review]
Plea

File: utils/theme.ts (hunk 2 above)
Comment (cupoftea4) [reply utils/theme.ts:43]: Everything before space kind of differs for light and dark, and everything after - doesn't, but might in the future

File: utils/theme.ts
Comment (cupoftea4) [reply utils/theme.ts:1]: i will move it in the next PR

File: utils/theme.ts (hunk 2 above)
Comment (Oleksii-Morozov) [reply utils/theme.ts:43]: Okay, but for future make it more understandable

**Review by Oleksii-Morozov — APPROVED** [review]

**Pull request merged**